ivy/functional/frontends/paddle/tensor/creation.py

```diff
@@ -76,3 +76,8 @@ def eye(num_rows, num_columns=None, dtype=None, name=None):
 @to_ivy_arrays_and_back
 def empty_like(x, dtype=None, name=None):
     return ivy.empty_like(x, dtype=dtype)
+
+
+@to_ivy_arrays_and_back
+def diagflat(x, offset=None, name=None):

```

Comment:
The default value of `offset` is `0` not `None`

Reply:
Oops!